lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -1113,6 +1128,13 @@ void _applyParagraphStyleToElement({
     if (style._fontFamily != previousStyle._fontFamily) {
       cssStyle.fontFamily = quoteFontFamily(style._fontFamily);
     }
+    if (style._shadows != previousStyle._shadows) {
```

Comment:
This covers the `else` case but not the `if` case.

Reply:
done.